Sources/PointFree/Auth.swift

```diff
@@ -62,14 +62,14 @@ extension URLRequest {
 
 public func readSessionCookieMiddleware<I, A>(
   _ conn: Conn<I, A>)
-  -> IO<Conn<I, Tuple2<A, Database.User?>>> {
+  -> IO<Conn<I, Tuple2<Database.User?, A>>> {
 
     let user = conn.request.cookies[pointFreeUserSession]
       .flatMap {
         ResponseHeader
           .verifiedString(signedCookieValue: $0, secret: AppEnvironment.current.envVars.appSecret)
       }
-      .flatMap(UUID.init >-> Database.User.Id.init)
+      .flatMap(UUID.init(uuidString:) >-> Database.User.Id.init)
```

Comment:
Did this become ambiguous?

Reply:
yeah cause I added `RawRepresentable` conformance to `UUID`